package.json

```diff
@@ -16,7 +16,9 @@
     "docs:json": "compodoc -p ./tsconfig.json -e json -d .",
     "storybook": "npm run docs:json && start-storybook -p 6006",
     "build-storybook": "npm run docs:json && build-storybook",
-    "extract-i18n": "ng extract-i18n --output-path ./src/locale/ && xliffmerge --profile xliffmerge.json"
+    "extract-i18n": "ng extract-i18n --output-path ./src/locale/ && xliffmerge --profile xliffmerge.json",
+    "cypress:open": "e2e open",
+    "cypress:run": "e2e run"
```

Comment:
what is the purpose of each of these two commands? I can't run them locally because `e2e` is not recognized. Instead I used `npm run e2e` which uses the `ng e2e` command which works quite fine.